### DISCUSSION TITLE
A treatment of an alternate ship boarding process that would make capture harder.

### DISCUSSION THREAD

**Discussion by DoublePants** [endless-sky/endless-sky#10120] [Ideas]
I did do some cursory searches for similar suggestions but I'm relatively inexperienced with Github so please forgive me if this is inappropriate, impossible, or redundant in some way and just close/delete if I missed something.

I know you have escape pods planned for future and those would be a great addition to this although not vital for this concept. I envision it like this by way of example, but any similar system would achieve the same end:

1. Docking is initiated as it currently functions. Only your ship's "extra crew" would board the enemy ship, however.
2. The boarded ship makes a "threat roll" against the boarding ship based on the size of the two ships and the size of their crews. This would determine the boarded ship's initial "stance" to being boarded. Initial stances would be: "Unconcerned" which represents a tiny ship boarding a much larger one. The boarded ship takes no actions against the boarders at all, being more concerned with damage control; this would be the best reaction if a boarder's plan is to steal vital components or do things like rescue captives if that's ever made a mission type. "Deploying Security" which represents minor concern; security teams are sent to oppose the boarding action, so only a percentage of the ship's crew plus its extra crew (and security stations) attempts to counter boarding actions initially. "Lockdown" would often occur when a larger ship boards a smaller one; in this stance the entire crew would attempt to counter actions, but would NOT attempt to counter if the boarders are only taking non-vital ship components (so they would only defend engines, power generation, and hyperdrives/jump drives). This would be the best result to get if one just wants to steal cargo or weapons. "Panic" would be the result of much larger ships boarding smaller ones, and would result in either an immediate surrender of the entire ship or an immediate abandon ship/self-destruct as soon as boarding is attempted. As of right now, human players being boarded would never be "Unconcerned," but if damage control ever becomes a concern in the game, then Unconcerned may become a viable selection. A human choosing "Panic" would essentially be a surrender/game over unless you ever add capture and ransom mechanics or escape pods.
3. At this point the boarding ship chooses an "Action," either seizing components or attempting to capture the ship. The odds of the action succeeding would depend on the action chosen versus the current reaction of the opposing ship. For example, seizing cargo or non-vital components would automatically succeed against a ship in Lockdown, and you would be highly unlikely to lose any crew in the process. The boarded ship would then react in the next step before the results are calculated.
4. After the attacking ship chooses its action, the defending ship chooses its own action, replacing its stance with a new set of stances based on their current goals. If the ship's initial stance was "Unconcerned," they will automatically pick the new stance "Surprised" for their first turn, which would have a very low chance of preventing the attacker's action and have a very low chance of killing any boarders. A "Deploying Security" stance would most often use a "Repel Boarders" stance, having a high chance of preventing the attacking Action (based on the size of the boarding force vs the size of the security team) and resulting in one round of the current combat system being executed, except that the boarded ship would only use a part of its crew and any installed security stations to fight back. A ship that chose "Lockdown" would stay in Lockdown unless the boarders attempted to seize a vital component, in which case they would switch to "Defend Ship," which would work the way current combat works using the entire ship's crew, but with a larger defender's advantage. A "Panic" ship would either explode as soon as it's docked with or would be treated as a ship with no crew and take no actions.
5. The result of step 4 would have the boarded ship make a new "threat roll," so it could go from being Unconcerned/Surprised to Deploying Security. If the ship switched to "Panic," it would either surrender or initiate the self-destruct sequence and all hands would abandon ship, which would give the boarding ship one more Action before the ship explodes (capture could be attempted but halting the self-destruct would have low odds of success and you'd lose crew if you failed). Military ships would be far more likely to self-destruct than surrender, whereas cargo ships would have more even odds of selecting between the two. A ship which saw a good result from a "Repel Boarders" Stance might select a new stance, "Counterattack" which would send the security force to attempt to capture the attacking ship. This would have a chance of switching the ship roles so that the boarder becomes the boarded. If a ship lost most of its security team on "Repel Boarders" it would likely switch to "Lockdown/Defend Ship" so it could start using its entire crew. A ship running out of crew in this paradigm would be likely to start its self-destruct sequence and abandon ship instead of fighting to the last man as the current system works.
6. Steps 3-5 repeat until the boarders leave the ship or the boarded ship's crew retreats in escape pods, surrenders, or is eliminated. If the crew is wiped out before the self-destruct is initiated, the ship would be captured the same way it works right now. Surrendering crew could be placed in a Brig if the attacking ship has one and ransomed/bountied in non-pirate lands or sold into slavery on pirate worlds.

My primary concerns in this concept were:

- Make it harder in general to capture ships without ruining the idea of being a pirate to raid for components, cargo, or supplies.
- Treat each crew member as a person instead of a number; the crew wants to live, they will not fight to the death and will attempt to escape or surrender instead (different races could have different weights on these choices!).
- Create a meaningful difference between warships and civilian ships with regards to how they react to boarding.
- Give security stations a more useful function, allowing boarding to be repelled without risking the entire crew or allowing vital components to be more easily defended.
- Make "raiding" a ship possible with a much smaller crew while still allowing ships to defend their ship components in a more fair fight.
- Make decision-making when boarding more important. Which ship you use to board, what you attempt to take, and how long to spend/how much to risk trying to take it should be a concern.
- Make ship combat feel like a contest instead of a numbers comparison.
- Allow players to achieve everything they can achieve with the current system, only introducing more challenges to get the best possible results.
- Focus on a process with selections which could be easily added to, removed, or modded.

Something like "ship status" could eventually be implemented here as well, representing things like self-destruct timers, "warp core breach"-like situations, and other crises which might affect the decision-making of the crews involved in a boarding attempt. Here, however, I wanted to focus exclusively on the Stance/Action concept.

Thank you for reading if you did. Endless Sky is a great project and I hope it becomes everything you envision!

**Comment by Zitchas** [comment]
That is an interesting idea. Thanks for elaborating it.
